src/test/java/org/example/Operations/CollectionOperationsTest.java

```diff
@@ -1,250 +1,486 @@
+            assertEquals("rods", name);
+            assertEquals("tempZone", zone);
+            assertEquals("rodsadmin", type);
+            assertEquals("read_object", perm);
+
+            // Remove rodsadmin's permission on the collection.
+            List<ModifyPermissionsOperations> operation2 = new ArrayList<>();
+            operation2.add(new ModifyPermissionsOperations("rods", "null"));
+            res = assertDoesNotThrow(() ->
+                            alice.collections().modify_permissions(aliceToken, collection, operation2, OptionalInt.empty()),
+                    "JsonProcessingException was thrown"
+            );
+
+            assertEquals("Modifying permissions on collection request failed", 200, res.getHttpStatusCode());
+            assertEquals("Modifying permissions on collection failed", 0,
+                    getIrodsResponseStatusCode(res.getBody()));
+
+            // Show the permissions have been removed.
+            Response statRes2 = alice.collections().stat(aliceToken, collection, Optional.empty());
+            assertEquals("Stat on collection request failed", 200, statRes2.getHttpStatusCode());
+            assertEquals("Stat on collection failed", 0,
+                    getIrodsResponseStatusCode(statRes2.getBody()));
+
+            rootNode = assertDoesNotThrow(() -> mapper.readTree(statRes2.getBody()),
+                    "JsonProcessingException was thrown");
+            int permissionArraySize = rootNode.path("permissions").size();
+            assertEquals(1, permissionArraySize);
+        } finally {
+            // Reset permissions
+            List<ModifyPermissionsOperations> operation2 = new ArrayList<>();
+            operation2.add(new ModifyPermissionsOperations("rods", "null"));
+            res = assertDoesNotThrow(() ->
+                            alice.collections().modify_permissions(aliceToken, collection, operation2, OptionalInt.empty()),
+                    "JsonProcessingException was thrown"
+            );
+        }
+    }
+
+    @Test
+    public void testTouchOperationUpdatesMtime() {
+        Response res;
+        String collection = "/tempZone/home/alice";
+
+        // Get the mtime of the home collection.
+        String query = "select COLL_MODIFY_TIME where COLL_NAME = '" + collection + "'";
+        Response queryRes = alice.queryOperations().execute_genquery(aliceToken, query, null);
+        assertEquals("Execute genquery request failed", 200, queryRes.getHttpStatusCode());
+        assertEquals("Execute genquery failed", 0,
+                getIrodsResponseStatusCode(queryRes.getBody()));
+
+        JsonNode rootNode = assertDoesNotThrow(() -> mapper.readTree(queryRes.getBody()),
+                "JsonProcessingException was thrown");
+        int rowsArraySize = rootNode.path("rows").size();
+        int originamMtime = rootNode.path("rows").get(0).get(0).asInt();
+
+        assertEquals(1, rowsArraySize);
+        assert(originamMtime > 0);
```

Comment:
There isn't an `assertGreater` or `assertTrue`?

_This applies to other uses of `assert` too._

Reply:
Changed to `assertTrue`.